detectors/nuxt.modules.json

```
@@ -394,5 +394,16 @@
     "detectors": {
       "js": "window.$nuxt?.$druxt"
     }
+  },
+  "@nuxt/nitro": {
+    "metas": {
+      "slug": "nuxt-nitro",
+      "name": "@nuxt/nitro",
+      "imgPath": null,
+      "url": "https://nitro.nuxtjs.org/"
```

```suggestion
      "url": "https://nuxtjs.org/"
```

(Docs will move to v3.nuxtjs.org. nitro subdomain is no used)